src/tech/data-standards.md

```diff
@@ -0,0 +1,45 @@
+# Data Standards at dxw
```

Comment:
Some of the Jekyll front matter is missing here. At a minimum:

```
---
title: Data standard at dxw
permalink: /:path/:basename/
---

```

Then the `h1` can go away (I sentence cased it in the example above because most of our playbook pages have sentence case titles)

Reply:
Good catch! Fixed.